pkg/testutils/serverutils/test_server_shim.go

```diff
@@ -224,7 +227,21 @@ func StartServer(
 	if err != nil {
 		t.Fatalf("%+v", err)
 	}
+
+	// Determine if we should probabilistically start tenants for the cluster.
+	const probabilityOfStartingTestTenant = 0.5
```

Comment:
Can we use the metamorphic constant thingie for this instead? That way, it gets logged at the top and is more generally discoverable.